modules/database/sqlite/activate.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Actions to run when the module gets activated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Copies the db.php file in wp-content and reloads the page.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	// Bail early if the SQLite3 class does not exist.
+	if ( ! class_exists( 'SQLite3' ) ) {
+		return;
+	}
+
+	$destination = WP_CONTENT_DIR . '/db.php';
+
+	// Bail early if the file already exists.
+	if ( file_exists( $destination ) ) {
+		return;
+	}
+
+	// Init the filesystem to allow copying the file.
+	global $wp_filesystem;
+	if ( ! $wp_filesystem ) {
+		require_once ABSPATH . '/wp-admin/includes/file.php';
+		WP_Filesystem();
+	}
+
+	// Copy the file, replacing contents as needed.
+	if ( $wp_filesystem->touch( $destination ) ) {
+
+		// Get the db.copy file contents, replace placeholders and write it to the destination.
+		$file_contents = str_replace(
+			array(
+				'{SQLITE_IMPLEMENTATION_FOLDER_PATH}',
+				'{PERFLAB_PLUGIN}',
+				'{SQLITE_MODULE}',
+				'{PERFLAB_MODULES_SETTING}',
+				'{PERFLAB_MODULES_SCREEN}',
+			),
+			array(
+				__DIR__,
+				str_replace( WP_PLUGIN_DIR . '/', '', PERFLAB_MAIN_FILE ),
```

Comment:
Yes, this works fine on Windows.

Reply:
Let's say on Windows the `PERFLAB_MAIN_FILE` constant is something like this:

```
C:\path\to\wordpress\wp-content\plugins\wordpress-performance\load.php
```

 Then `WP_PLUGIN_DIR . '/'` will be `C:\path\to\wordpress\wp-content\plugins/` and the `str_replace` function won't be able to find it in the `$subject`, thus it will return the same string without stripping the plugins dir path.

Reply:
```suggestion
				str_replace( WP_PLUGIN_DIR . DIRECTORY_SEPARATOR, '', PERFLAB_MAIN_FILE ),
```

Reply:
Thank you for the detailed explanation @eugene-manuilov !

Makes sense now... Committed your fix 👍 🙇